wrappers/node/src/api/common.ts

```diff
@@ -165,10 +165,6 @@ export enum VerifierStateType {
   Failed = 5
 }
 
-export interface IInitVCXOptions {
-  libVCXPath?: string;
-}
-
 export interface ISerializedData<T> {
```

Comment:
Note: we agreed to remove the assumptions this interface makes about the serialization format (i.e. containing `source_id`, `data`, `version` or any other fields, but the assumption of it being a JSON will stay for now).